apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -266,13 +261,14 @@ collect_total_from_tail_nodes(Nodes, QueryState, ResultAcc = #{total := TotalAcc
 %%--------------------------------------------------------------------
 
 %% QueryState ::
```

Comment:
Unrelated to this PR.  I wonder if this could be a proper typespec and have dialyzer check it.  :thinking:

Reply:
Certainly. There's a followup planned where we want to redesign these APIs quite a bit. There I plan to refactor this code substantially and spice it up with typespecs.